### PR TITLE
Create simpson.cpp

### PR DESCRIPTION
**Pull request by kichloo**
Added the implementation of Simpson's 1/3 rule in C language.

**Fixes issue:**
Added another rule (Simpson's 1/3 rule) for integration.

**Changes:**
Added the file simpson.cpp

### CODE REVIEW

File: simpson.cpp [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#include<stdio.h>
+#include<conio.h>
```
Comment (arnavb) [inline simpson.cpp]: Don't use this. Reformat this file using clang-format with 4 spaces. Place this file into the correct directory.

File: simpson.cpp [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+float f(float x)
+{
+    return(1/(1+x));
```
Comment (arnavb) [inline simpson.cpp]: No need for extra parentheses on the outside of the expression.

File: simpson.cpp [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+}
+void main()
```
Comment (arnavb) [inline simpson.cpp]:
```suggestion
int main()
```

File: simpson.cpp [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+{
+    int i,n;
```
Comment (arnavb) [inline simpson.cpp]: Declare i in the loop itself.

File: simpson.cpp [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    float x0,xn,h,y[20],so,se,ans,x[20];
```
Comment (arnavb) [inline simpson.cpp]: Declare these variables when they are first used, not at the top of the function.

File: simpson.cpp [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+    scanf("%f%f%f",&x0,&xn,&h);
+    n=(xn-x0)/h;
+    if(n%2==1)
+    {
+        n=n+1;
+    }
+    h=(xn-x0)/n;
+    printf("\n Refined value of n and h are:%d %f\n",n,h);
+    printf("\n Y values: \n");
+    for(i=0; i<=n; i++)
```
Comment (arnavb) [inline simpson.cpp]: Pre-increment in all for loops.

File: simpson.cpp [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+            so=so+y[i];
+        }
+        else
+        {
+            se=se+y[i];
+        }
+ 
+    }
+    ans=h/3*(y[0]+y[n]+4*so+2*se);
+    printf("\n Final integration is %f",ans);
```
Comment (arnavb) [inline simpson.cpp]: Use `cout` and `cin` in C++.

File: simpson.cpp (hunk 1 above)
Comment (kichloo) [reply simpson.cpp]: @arnavb This is my first time contributing here. So, could you help regarding the correct directory?

**Comment by kichloo** [comment]
@arnavb Please check #4774 as I did it with the required changes

**Pull request closed without merging**